docs/reference/nuspec.md

```diff
@@ -368,15 +368,17 @@ The following example shows different variations of the `<group>` element:
         <dependency id="RouteMagic" version="1.1.0" />
     </group>
 
-    <group targetFramework="net40">
+    <group targetFramework=".NETFramework4.7.2">
         <dependency id="jQuery" version="1.6.2" />
         <dependency id="WebActivator" version="1.4.4" />
     </group>
 
-    <group targetFramework="sl30">
+    <group targetFramework="Silverlight4.0">
     </group>
 </dependencies>
 ```
+> [!Note]
+> The format of TFM identifier used in `assembly paths` is different when compared to the TFM identifier used in `dependency groups`. If the target frameworks declared in the dependencies group of the `nuspec` and the `lib/ref` folder do not have exact matches then `pack` command will raise [NuGet Warning NU5128](../reference/errors-and-warnings/nu5128).
```

Comment:
It would be nice to have a link here to the list of valid assembly path TFMs

Reply:
I have exactly the same feedback. As per my discussion with @nkolev92, it looks like TFM identifiers are not well documented and hence using [NuGet's MSBuild pack target](https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets) is an ideal solution. The solution is well documented in [NuGet Warning NU5128](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128).